### ISSUE TITLE
Option to hide "off"?

### ISSUE THREAD

**Issue by nibblerrick** [finity69x2/fan-percent-button-row#12]
Thanks for your great preset-row-cards!
I am using it at the moment to overcome the not-so-great-knx-climate-integration which lacks different features as swing and fan mode in the climate-entity, but now there is "fan" in knx, so I can add the fan of the climate.entity standalone and put your preset-row under the climate-entity...
So, problem is, the knx-fan-integration thinks there is an "off"-feature, but the acutal climate-entity hasn't something like "off" for the fan.
So long story short: I would like to hide the "off" button.
If it isn't something which can be integrated easily or is a request which noone than me would ever use it's fine to close the issue.
Of course it would be better to have the correct knx-integration, but it took quite some time until at least "fan" came along.

**Comment by finity69x2** [comment]
I doubt I would add that option.

it makes sense for other things like hiding some infrequently used buttons but I think the "off" function is an inherent part of most fans.

Sorry about that.

**Comment by nibblerrick** [comment]
Ok, no problem, thanks for taking the time to answer.
Maybe I try to hide it with card-mod when it bothers me to much. However, it is very convenient to have your card to make this workaround, even if "off" has no function in this case.

**Comment by nibblerrick** [comment]
Hey, thanks for implementing that now :-)
